src/util.js

```diff
@@ -271,7 +271,7 @@ function timeif(bool, name, thunk) {
 }
 
 function pipeline(fns) {
-  return _.compose.apply(null, fns.reverse());
+  return _.flowRight.apply(null, fns.reverse());
```

Comment:
`_.flow` invokes functions left to right, whereas `_.compose`and `_.flowRight` is a normal function composition you'd expect where the last argument is the innermost fn, etc. A bit confusing, take a look here: https://lodash.com/docs/4.17.2#flow